Modules/ITS/src/ITSRawTask.cxx

```diff
@@ -0,0 +1,771 @@
+
+  h->SetStats(0);
+
+
+}
+/*
+void ITSRawTask::format2DZaxis(TH2 *h){
+
+  TPaletteAxis *palette = (TPaletteAxis*)h->GetListOfFunctions()->FindObject("palette");
+  palette->SetLabelSize(0.5);
+ // TPad *pad = ()
+
+}
+*/
+void ITSRawTask::formatAxes(TH1 *h, const char* xTitle, const char* yTitle, float xOffset, float yOffset)
+{
+  h->GetXaxis()->SetTitle(xTitle);
+  h->GetYaxis()->SetTitle(yTitle);
+  h->GetXaxis()->SetTitleOffset(xOffset);
+  h->GetYaxis()->SetTitleOffset(yOffset);
+}
+
+void ITSRawTask::formatPaveText(TPaveText *aPT, float aTextSize, Color_t aTextColor, short aTextAlign, const char *aText)
+{
+  aPT->SetTextSize(aTextSize);
+  aPT->SetTextAlign(aTextAlign);
+  aPT->SetFillColor(0);
+  aPT->SetTextColor(aTextColor);
+  aPT->AddText(aText);
+}
+
+void ITSRawTask::ConfirmXAxis(TH1 *h)
+{
+  // Remove the current axis
+  h->GetXaxis()->SetLabelOffset(999);
+  h->GetXaxis()->SetTickLength(0);
+  // Redraw the new axis
+  gPad->Update();
+  int XTicks = (h->GetXaxis()->GetXmax() - h->GetXaxis()->GetXmin()) / DivisionStep;
+
+  TGaxis *newaxis = new TGaxis(gPad->GetUxmin(), gPad->GetUymin(), gPad->GetUxmax(), gPad->GetUymin(),
+      h->GetXaxis()->GetXmin(), h->GetXaxis()->GetXmax(), XTicks, "N");
+  newaxis->SetLabelOffset(0.0);
+  newaxis->Draw();
+  h->GetListOfFunctions()->Add(newaxis);
+}
+
+void ITSRawTask::ReverseYAxis(TH1 *h)
+{
+  // Remove the current axis
+  h->GetYaxis()->SetLabelOffset(999);
+  h->GetYaxis()->SetTickLength(0);
+
+  // Redraw the new axis
+  gPad->Update();
+
+  int YTicks = (h->GetYaxis()->GetXmax() - h->GetYaxis()->GetXmin()) / DivisionStep;
+  TGaxis *newaxis = new TGaxis(gPad->GetUxmin(), gPad->GetUymax(), gPad->GetUxmin() - 0.001, gPad->GetUymin(),
+      h->GetYaxis()->GetXmin(), h->GetYaxis()->GetXmax(), YTicks, "N");
+
+  newaxis->SetLabelOffset(0);
+  newaxis->Draw();
+  h->GetListOfFunctions()->Add(newaxis);
+
+}
+
+void ITSRawTask::publishHistos()
+{
+  for (unsigned int iObj = 0; iObj < m_publishedObjects.size(); iObj++) {
+    getObjectsManager()->startPublishing(m_publishedObjects.at(iObj));
+  }
+}
+
+void ITSRawTask::addMetadata(int runID, int EpID, int fileID)
+{
+  for (unsigned int iObj = 0; iObj < m_publishedObjects.size(); iObj++) {
+    getObjectsManager()->addMetadata(m_publishedObjects.at(iObj)->GetName(), "Run", Form("%d", runID));
+    getObjectsManager()->addMetadata(m_publishedObjects.at(iObj)->GetName(), "EP", Form("%d", EpID));
+    getObjectsManager()->addMetadata(m_publishedObjects.at(iObj)->GetName(), "File", Form("%d", fileID));
+  }
+}
+
+void ITSRawTask::getProcessStatus(int aInfoFile, int& aFileFinish)
+{
+  aFileFinish = aInfoFile % 10;
+  //cout<<"aInfoFile = "<<aInfoFile<<endl;
+  FileRest = (aInfoFile - aFileFinish) / 10;
+
+  QcInfoLogger::GetInstance() << "FileFinish = " << aFileFinish << AliceO2::InfoLogger::InfoLogger::endm;
+  QcInfoLogger::GetInstance() << "FileRest = " << FileRest << AliceO2::InfoLogger::InfoLogger::endm;
+
+  if (aFileFinish == 0)
```

Comment:
Please use brackets also for if statements with just one line, following the coding guidelines https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Braces#Braces

Reply:
It's done.